src/redux/actions/common.js

```diff
@@ -36,9 +37,28 @@ export const clearError = () => ({
  */
 export const serverError = () => ({
   type: SERVER_ERROR,
-  error: "Oops something went wrong"
+  error: { serverError: "Oops something went wrong!!" }
```

Comment:
This can be formatted to:
*type: SERVER_ERROR,
error: 
    {
        serverError: "Oops ..."
    }* to improve its readability

Reply:
@meshack-mbuvi @kimotho-njoki linter will not allow it that way as it has not reached the maximum line length